samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/{{cookiecutter.project_name}}/README.md

```diff
@@ -125,48 +124,52 @@ build.ps1 --target=Test
-    --capabilities CAPABILITY_IAM \
-    --parameter-overrides MyParameterSample=MySampleValue
+* Create a catch all resource (e.g. /hello/{proxy+}) and return the name requested through this new path
+* Update tests
 
-aws cloudformation describe-stacks \
-    --stack-name {{ cookiecutter.project_name.lower().replace(' ', '-') }} --query 'Stacks[].Outputs'
-```
+### Step-through debugging
 
-## Next Steps
+* **[Enable step-through debugging docs for supported runtimes](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-using-debugging.html)**
 
-Create your own .NET Core solution template to use with SAM CLI. [Cookiecutter for AWS SAM and .NET](https://github.com/aws-samples/cookiecutter-aws-sam-dotnet) provides you with a sample implementation how to use cookiecutter templating library to standardise how you initialise your Serverless projects.
+Next, you can use AWS Serverless Application Repository to deploy ready to use Apps that go beyond hello world samples and learn how authors developed their applications: [AWS Serverless Application Repository main page](https://aws.amazon.com/serverless/serverlessrepo/)
 
-``` bash
- sam init --location gh:aws-samples/cookiecutter-aws-sam-dotnet
-```
+## SAM and AWS CLI commands
 
-For more information and examples of how to use `sam init` run 
+All commands used throughout this document
 
-``` bash
-sam init --help
-```
+```bash
+# Invoke function locally with event.json as an input
+sam local invoke HelloWorldFunction --event event.json
 
-## Bringing to the next level
+# Run API Gateway locally
+sam local start-api
 
-Here are a few ideas that you can use to get more acquainted as to how this overall process works:
+# Create S3 bucket
+aws s3 mb s3://BUCKET_NAME
 
-* Create an additional API resource (e.g. /hello/{proxy+}) and return the name requested through this new path
-* Update unit test to capture that
-* Package & Deploy
+# Package Lambda function defined locally and upload to S3 as an artifact
+sam package \
+    --output-template-file packaged.yaml \
+    --s3-bucket REPLACE_THIS_WITH_YOUR_S3_BUCKET_NAME
 
-Next, you can use the following resources to know more about beyond hello world samples and how others structure their Serverless applications:
+# Deploy SAM template as a CloudFormation stack
+sam deploy \
+    --template-file packaged.yaml \
+    --stack-name {{ cookiecutter.project_name.lower().replace(' ', '-') }} \
+    --capabilities CAPABILITY_IAM
 
-* [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)
+# Describe Output section of CloudFormation stack previously created
+aws cloudformation describe-stacks \
+    --stack-name {{ cookiecutter.project_name.lower().replace(' ', '-') }} \
+    --query 'Stacks[].Outputs[?OutputKey==`HelloWorldApi`]' \
+    --output table
+
+# Tail Lambda function Logs using Logical name defined in SAM Template
```

Comment:
Should we have a step to invoke the function or API gateway locally with `sam logs` running in the terminal?